src/main/java/ovh/akio/hmu/Utils.java

```diff
@@ -114,12 +114,21 @@ public static List<PckAudioFile> scanPck(File path) {
             return Collections.emptyList();
         }
 
-        return Utils.getDirectoryContent(path)
+        if (!Application.MAIN.AllowAnyAudioFiles) {
```

Comment:
I successfully utilized this modification to unpack sfx-related resources, but there is currently an issue with the code: the condition here is actually "unpack all resources when --all is not specified," which is contrary to the described behavior.